BREAKING_CHANGES.md

```diff
@@ -0,0 +1,25 @@
+## Breaking Changes in Terraform provider
+
+## Version Compatibilities
+
+Our releases follow the Semantic Versioning (SemVer) convention to ensure predictability and stability. Each version is represented as MAJOR.MINOR.PATCH, where:
+
+- MAJOR version updates may contain breaking changes.
+- MINOR version updates introduce backward-compatible functionality and can contain breaking changes 
+- PATCH version updates encompass backward-compatible bug fixes.
```

Comment:
I think for both versioning & changelog, it would be more accurate to refer to the Terraform guidance/policy, which is what we aim to follow: https://developer.hashicorp.com/terraform/plugin/best-practices/versioning#versioning-specification